src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -73,6 +83,10 @@ protected FieldBuilder(FieldType fieldType)
             return new FieldBuilder<TSourceType, TReturnType>(fieldType);
         }
 
+        //private static void Reference<T>()
```

Comment:
?

Reply:
I'll clean it up.  AOT uses analysis of the generated IL in order to determine which types are eligible for trimming.  Sometimes tricks like this one can be used to keep a specified type 'alive'.  During JIT compilation it would be removed so it should not affect speed for other users.  Not to mention this code is only used during schema initialization.